### DISCUSSION TITLE
How change value without output yaml ?

### DISCUSSION THREAD

**Discussion by patsevanton** [mikefarah/yq#1459] [Q&A]
Hello! Thanks for yq

I run
`yq -i '.a.b[0].c = "cool"' file.yaml`
I see output yaml.

How change value without output yaml or may be silent/quiet mode?

**Comment by mikefarah** [comment]
Which version are you using - I can't replicate that

**Comment by patsevanton** [comment]
I am sorry. I changed without -i
yq -i '.a.b[0].c = "cool"' file.yaml work
